cluster/addons/prometheus/alertmanager-deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: alertmanager
```

Comment:
This is another reason why the Prometheus Operator is a good idea here. Running the alertmanager as a deployment is a bad idea, it delivers notifications about alerts and therefore needs to be the most reliable thing in your cluster. With a simple `Deployment` you're not getting any of the high availability features that you want, such as persistent storage and controlled rolling deployments.